src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -945,18 +974,89 @@ private int indexDown(File dir, String parent, boolean count_only,
                             continue;
                         }
                     }
-                    try {
-                        addFile(file, path);
-                    } catch (Exception e) {
-                        LOGGER.log(Level.WARNING,
-                                "Failed to add file " + file.getAbsolutePath(),
-                                e);
-                    }
+
+                    args.works.add(new IndexFileWork(file, path));
                 }
             }
         }
+    }
 
-        return lcur_count;
+    /**
+     * Executes the second, parallel stage of indexing.
+     * @param args contains a list of files to index, found during the earlier
+     * stage
+     */
+    private void indexParallel(IndexDownArgs args) throws IOException {
+
+        int worksCount = args.works.size();
+        if (worksCount < 1) return;
```

Comment:
some log message?

Reply:
There are already messages about number of files updated/deleted and total.

Reply:
ok